schema/schema-linux.json

```diff
@@ -0,0 +1,291 @@
+                             "$ref": "defs.json#/definitions/uint64Pointer"
+                         },
+                         "swap": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/resources/memory/swap",
+                             "$ref": "defs.json#/definitions/uint64Pointer"
+                         },
+                         "swappiness": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/resources/memory/swappiness",
+                             "$ref": "defs.json#/definitions/uint64Pointer"
+                         }
+                     }
+                 },
+                 "network": {
+                     "id": "https://www.opencontainers.org/jsonschema/linux/resources/network",
+                     "type": "object",
+                     "properties": {
+                         "classId": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/resources/network/classId",
+                             "type": "string"
+                         },
+                         "priorities": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/resources/network/priorities",
+                             "oneOf": [
+                               {
+                                 "type": "array",
+                                 "items": {
+                                   "oneOf": [ { "$ref": "defs-linux.json#/definitions/NetworkInterfacePriority" } ]
+                                 }
+                               },
+                               { "type": "null" }
+                             ]
+                         }
+                     }
+                 }
+             }
+         },
+         "rlimits": {
+             "id": "https://www.opencontainers.org/jsonschema/linux/rlimits",
+             "items": [
+                 {
+                     "id": "https://www.opencontainers.org/jsonschema/linux/rlimits/0",
+                     "properties": {
+                         "hard": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/rlimits/0/hard",
+                             "type": "integer"
+                         },
+                         "soft": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/rlimits/0/soft",
+                             "type": "integer"
+                         },
+                         "type": {
+                             "id": "https://www.opencontainers.org/jsonschema/linux/rlimits/0/type",
+                             "type": "integer"
+                         }
+                     },
+                     "type": "object"
+                 }
+             ],
+             "type": "array"
+         },
+         "cgroupsPath": {
+         "oneOf": [
```

Comment:
Never cared to used `jq`, and looking at it now, it returns a compile error for `./schema.json`, where as it is a valid doc.
Occasionally I'll check against `python -mjson.tool` because it's on most every host without needing additional packages. Though I don't use it for formatting as it rearranges as the values.